http-clients/build.gradle

```diff
@@ -21,6 +21,7 @@ dependencies {
     compile libs.wsRsApi
 
     testCompile project(':http-servers')
+    testCompile files('lib/VersionTest.jar')
```

Comment:
tried that with SerializableError but since it's not packaged before the test runs you end up getting null back for the version and name